apps/papra-server/Dockerfile

```
@@ -56,5 +56,9 @@ COPY pnpm-workspace.yaml ./
 
 EXPOSE 1221
 
+# Validate /api/health (node is in the image; avoids curl/jq on slim)
+HEALTHCHECK --interval=30s --timeout=5s --start-period=20s --retries=3 \
```

Hadolint (2.15.1)

[warning] Use arguments JSON notation for CMD and ENTRYPOINT arguments

(DL3025)